src/base/unicodestrings.h

```diff
@@ -35,6 +35,7 @@
 // we put all problematic UTF-8 chars/strings in this file.
 // See issue #3059 for more details (https://github.com/qbittorrent/qBittorrent/issues/3059).
 const char C_INFINITY[] = "∞";
+const char C_NON_BREAKING_SPACE[] = " ";
```

Comment:
Take a note of this I think you did something similar in "AddNewTorrent" dialog when you refactored it. Could this be useful there too?